bin/wmcore-db-init

```diff
@@ -121,7 +128,18 @@ def create(config):
     if hasattr(config.CoreDatabase, "indexspaceName"):
         params["tablespace_index"] = config.CoreDatabase.indexspaceName
 
-    wmInit.setSchema(modules, params = params) 
+    wmInit.setSchema(modules, params = params)
+    return
+
+def destroy(config):
+    params = {}
+
+    if hasattr(config.CoreDatabase, "tablespaceName"):
+        params["tablespace_table"] = config.CoreDatabase.tablespaceName
+    if hasattr(config.CoreDatabase, "indexspaceName"):
+        params["tablespace_index"] = config.CoreDatabase.indexspaceName
+
```

Comment:
Ops, I forgot to remove it! Done

On Wed, Jul 8, 2015 at 6:17 PM, ticoann <notifications@github.com> wrote:

> In bin/wmcore-db-init
> <https://github.com/dmwm/WMCore/pull/6055#discussion_r34167169>:
>
> > @@ -121,7 +128,18 @@ def create(config):
> >      if hasattr(config.CoreDatabase, "indexspaceName"):
> >          params["tablespace_index"] = config.CoreDatabase.indexspaceName
> >
> > -    wmInit.setSchema(modules, params = params)
> > +    wmInit.setSchema(modules, params = params)
> > +    return
> > +
> > +def destroy(config):
> > +    params = {}
> > +
> > +    if hasattr(config.CoreDatabase, "tablespaceName"):
> > +        params["tablespace_table"] = config.CoreDatabase.tablespaceName
> > +    if hasattr(config.CoreDatabase, "indexspaceName"):
> > +        params["tablespace_index"] = config.CoreDatabase.indexspaceName
> > +
>
> Is setting 'params' needed? It doesn't seems to be used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dmwm/WMCore/pull/6055/files#r34167169>.
>